scripts/cli/util/util.py

```diff
@@ -0,0 +1,148 @@
+import copy
+import os
+import socket
+import subprocess
+import time
+
+from os.path import join
+
+def set_environ():
+  '''This function inserts the necessary environment variables to the os.environ
+  dictionary and returns the old one so that can be restored later.'''
+  old = copy.copy(os.environ)
+
+  cc = os.environ['CODECOMPASS_HOME']
+
+  os.environ['SASL_PATH'] = join(cc, 'lib', 'sasl2')
```

Comment:
http://asg.web.cmu.edu/sasl/
We removed the authentication for now so I remove this line too.